docs/guide/workflow/workflow.md

```diff
@@ -76,6 +76,16 @@ The notification is displayed in the user menu:
 
 ![Notification about content to review](../img/workflow_notification.png)
 
+#### Draft locking
+
+You can enable draft locking, so that when a user sends a draft to review, only 
+the assigned reviewer can either edit the draft or unlock it for editing, and no 
+other user can take it over. 
+
+Use the [Version Lock Limitation](../limitation_reference.md#version-lock-limitation) together with the `content/edit` and 
+`content/unlock` Policies to enable locking drafts to specific users and prevent 
```

Comment:
This is not about enabling locking. User will assign himself to the content (and lock it) regardless of this policy. It will just prevent (or not, depending on limitation setting) editing the content.